code/datums/traits/neutral.dm

```diff
@@ -240,3 +240,25 @@
 ///Applies a bad moodlet for having an uncovered head
 /datum/quirk/bald/proc/unequip_hat(mob/user, obj/item/clothing, force, newloc, no_move, invdrop, silent)
 	SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_hair_day", /datum/mood_event/bald)
+
+/datum/quirk/longtimer
+	name = "Longtimer"
+	desc = "You've been around for a long time and seen more than your fair share of action, suffering some pretty nasty scars along the way. For whatever reason, you've declined to get them removed or augmented."
+	value = 0
+	gain_text = "<span class='notice'>Your body has seen better days.</span>"
+	lose_text = "<span class='notice'>Your sins may wash away, but those scars are here to stay...</span>"
```

Comment:
... except, they're not, because you just lost the quirk

Reply:
I'm not gonna separately track and remove the scars just in case you somehow lose the quirk when there's no legitimate way to lose quirks while alive in the first place

Reply:
I'm not gonna separately track and remove the scars just in case you somehow lose the quirk when there's no legitimate way to lose quirks while alive in the first place